shared/src/ledger/wallet/store.rs

```diff
@@ -504,6 +504,15 @@ impl Store {
         address: Address,
         force: bool,
     ) -> Option<Alias> {
+        // abort if the address already exists in the wallet
+        if self.addresses.contains_right(&address) {
```

Comment:
I think this should respect the `force` flag

Reply:
Thanks! Fixed that.